examples/cinder-csi-plugin/snapshot/example.yaml

```diff
@@ -11,6 +11,8 @@ kind: VolumeSnapshotClass
 metadata:
   name: csi-cinder-snapclass
 snapshotter: cinder.csi.openstack.org
+parameters:
+  snapshot-inuse-volumes: "false"
```

Comment:
As suggested in my previous review, I suggest we use `force-create: "false"` for brevity.